### PR TITLE
Feature/refactoring

### LINKED ISSUE #41: Refactor role to include new Molecule testing scenarios, etc.

### PR DESCRIPTION
**Pull request by mrlesmithjr**
Closes #41

### CODE REVIEW

**Comment by elcomtik** [comment]
@nertwork can you please look at this? we would appreciate your review.

**Comment by elcomtik** [comment]
@mrlesmithjr can we move forward without a review from @nertwork ?

**Comment by nertwork** [comment]
@mrlesmithjr @elcomtik - Will look at this and review in detail by the end of the week.

File: .travis.yml [hunk 1]
```diff
@@ -1,19 +1,26 @@
...
-  - "3.6"
-
 services:
   - docker
-
+before_install:
+  - sudo apt-get -qq update
+env:
+  - molecule_distro=centos7
+  # - molecule_distro=centos8
```
Comment (nertwork) [inline .travis.yml:10]: Are these not tested - I see molecule files for them?

File: .travis.yml [hunk 2]
```diff
@@ -1,19 +1,26 @@
...
+  # - molecule_distro=debian8
+  # - molecule_distro=debian9
+  # - molecule_distro=debian10
+  # - molecule_distro=fedora
+  - molecule_distro=ubuntu1604
+  - molecule_distro=ubuntu1804
+  # - molecule_distro=ubuntu2004
```
Comment (nertwork) [inline .travis.yml:17]: Is focal not being tested? I see molecule files for them?

Comment (mrlesmithjr) [reply .travis.yml:17]: I believe they were failing for some reason. I need to review again.

Comment (mrlesmithjr) [reply .travis.yml:17]: https://github.com/mrlesmithjr/ansible-frr/runs/971531291?check_suite_focus=true#step:5:131

Comment (nertwork) [reply .travis.yml:17]: Okay let's get this merged for now - I'll look at some of the testing later on 👍

Comment (mrlesmithjr) [reply .travis.yml:17]: Merging now!

**Pull request merged**